src/nilss.jl

```diff
@@ -262,7 +262,7 @@ function forward_sense(prob::NILSSProblem,nilss::NILSS,alg)
   for iseg=1:nseg
     # compute y, w, vstar
     # _sol is a numindvar*(1+nus+1) x nstep matrix
-    _sol = Array(solve(_prob, alg, saveat=dtsave, dt=dtsave)(_prob.tspan[1]:dtsave:_prob.tspan[2]))
+    _sol = Array(solve(_prob, alg, saveat=dtsave))
```

Comment:
IIRC this addition was necessary because if I ran NILSS many times, there could be endpoint problems (the saved states (`v`, `w`) are preallocated, and then there was a size mismatch).